src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectFactory.cs

```diff
@@ -44,12 +50,18 @@ public VisualStudioProject CreateAndAddToWorkspace(string projectSystemName, str
             => CreateAndAddToWorkspace(projectSystemName, language, new VisualStudioProjectCreationInfo());
 
         public VisualStudioProject CreateAndAddToWorkspace(string projectSystemName, string language, VisualStudioProjectCreationInfo creationInfo)
+            => _threadingContext.JoinableTaskFactory.Run(async () => await CreateAndAddToWorkspaceAsync(
+                projectSystemName, language, creationInfo, CancellationToken.None).ConfigureAwait(false));
```

Comment:
The ConfigureAwait(false) isn't needed here. Just don't use an async lambda and that should tidy this up.

Reply:
dne.